core/src/test/java/io/questdb/test/griffin/SqlParserTest.java

```diff
@@ -56,7 +56,91 @@
 import static io.questdb.test.tools.TestUtils.getSystemTablesCount;
 
 public class SqlParserTest extends AbstractSqlParserTest {
+    private static final String aaplDdl = "CREATE TABLE 'AAPL_orderbook' (\n" +
```

Comment:
nit, please move DECLARE tests into separate class, SqlParserTest is 10k lines already and there is nothing it does that superclass cannot do